packages/marshal/src/make-far.js

```diff
@@ -81,8 +83,8 @@ const assertCanBeRemotable = candidate =>
  * Carol's `iface` as misrepresented by VatA.
  * @param {undefined} [props=undefined] Currently may only be undefined.
  * That plan is that own-properties are copied to the remotable
- * @param {object} [remotable={}] The object used as the remotable
- * @returns {object} remotable, modified for debuggability
+ * @param {any} [remotable={}] The object used as the remotable
+ * @returns {any} remotable, modified for debuggability
```

Comment:
This was necessary to fix a type inference issue in `zcfZygote`: https://github.com/Agoric/agoric-sdk/blob/mfig-marshal-explicit-jsdoc/packages/zoe/src/contractFacet/zcfZygote.js#L242

The inferred type when parsing JS was actually `any`, but somehow when switching to emitted declaration, it carried the `object` type, which is incompatible in some usages.

Reply:
Unfortunate to regress in type specificity but solving it would be a rabbit hole that's out of scope.